youths/tests/test_youth_profiles_graphql_api.py

```diff
@@ -168,6 +169,44 @@ def test_normal_user_can_create_youth_profile_mutation(rf, user_gql_client):
     assert dict(executed["data"]["createYouthProfile"]) == expected_data
 
 
+def test_superuser_cannot_create_youth_profile_mutation_for_others_profile(
```

Comment:
but isn't it supposed to work eventually? I mean, at some point superusers (or just admins with proper permissions) SHOULD be able to do all of these things? so these tests will be removed then? 🙂 and it should happen relatively soon, I guess, as venepaikka would need at least updating profiles by the admins.

Reply:
Maybe I should fix those tests. The purpose was to write tests where we check that no one without either explicit permissions or being the owner of the resource can't do the mutation. The plan was not to remove those test but write new ones for service specific admins. Maybe I should add the support for `serviceType` specific permissions and rewrite the tests since I admit, they are currently a bit misleading.